Misc/NEWS.d/next/Library/2023-01-05-23-04-15.gh-issue-91851.AuCzU5.rst

```diff
@@ -0,0 +1,3 @@
+Microoptimizations for :func:`fractions.Fraction.__round__`,
```

Comment:
I think these should all use the `:meth:` role rather than the `:func:` role

Reply:
Changed.  (I wasn't sure this pr require a release notes entry.)

Reply:
We generally add news entries for all changes that might be visible to users. A 2x speed increase (even if it's just for some very specific fraction operations) definitely counts!